demo/src/plugins/dashboard/components/top-articles/index.js

```diff
@@ -33,18 +34,24 @@ export default function TopArticles(props) {
             <tr>
               <th>Title</th>
               <th className="px-3"></th>
+              <th></th>
               <th>Slotting</th>
               <th>Order Date</th>
               <th></th>
             </tr>
             </thead>
             <tbody>
             {response.get('nodes', []).map(node => {
+              const isLocked = node.get('is_locked');
+              if (isLocked && !canUnlock) {
+                return;
+              }
               const handleRowClick = createRowClickHandler(navigate, node);
               return (
                 <tr key={`${node.get('_id')}`} className={`status-${node.get('status')} cursor-pointer`} onClick={handleRowClick}>
                   <td className="td-title">{node.get('title')}</td>
                   <td className="text-nowrap px-1 py-1"><Collaborators nodeRef={node.generateNodeRef().toString()} /></td>
+                  <td>{isLocked && (<Icon imgSrc="locked" alt="locked" />)}</td>
```

Comment:
@mtakla16 move the lock to the title td (before title) so we don't need a new th/td